staramr/results/QualityModule.py

```diff
@@ -0,0 +1,284 @@
+            num_contigs = 0
+
+            for contig in file_contigs_lengths:
+                if contig >= minimum_contig_length:
+                    num_contigs = num_contigs+1
+
+            file_num_contigs.append(num_contigs)
+
+
+        for file_num_contigs_over_minimum_bp in file_num_contigs:
+            if file_num_contigs_over_minimum_bp >= unacceptable_num_contigs_over_minimum_bp:
+                contigs_over_minimum_bp_feedback.append(False)
+
+            else:
+                contigs_over_minimum_bp_feedback.append(True)
+
+
+        feedback.append(file_num_contigs)
+        feedback.append(contigs_over_minimum_bp_feedback)
+
+        return feedback
+
+    def _get_quality_module(self,genome_length_feedback,N50_feedback,contigs_over_minimum_bp_feedback):
+        """
+        Goes through the files and for each provides detailed feedback for why they failed the quality metrics
+        :param genome_length_feedback: An array where each element is the feedback (true or false) for the corresponding file in regards to the
+        genome length quality metric
+        :param N50_feedback: An array where each element is the feedback (true or false) for the corresponding file in regards to the
+        the N50 quality metric
+        :param contigs_over_minimum_bp_feedback: An array where each element is the feedback (true or false) for the corresponding file in regards to the
+        the acceptable number of contigs equal to or above the minimum contig length quality metric
+        :return: An array where the first element is itself an array where each element is the detailed quality metric feedback for
+        the corresponding file. The second element is itself an array where each element is the feedback (true or false) 
+        for whether the corresponding file passes all of the quality metrics
+        """
+        #This array is what we will return and will contain the files feedback for whether or not they pass the quality metrics as well as the specific metrics
+        #the files fail on, if they fail any of the metrics and will be used to construct our quality module dataframe
+        feedback = []
+        #This array contains the feedback of either True or False for whether the corresponding file passes all the quality metrics 
+        #it will be returned as the first element of feedback
+        quality_parameter = []
+        #This array contains the detailed feeback or either nothing if the corresponding file passes all the quality metrics or the metrics which the corresponding file
+        #fails on and it will be returned as the second element of feedback
+        quality_parameter_feedback = []
+        for file_genome_length_feedback,file_N50_feedback,file_contigs_over_minimum_bp_feedback in zip(genome_length_feedback,N50_feedback,contigs_over_minimum_bp_feedback):
+            if file_genome_length_feedback == True & file_N50_feedback == True & file_contigs_over_minimum_bp_feedback == True:
```

Comment:
Switch this long & statement to use [python all()](https://docs.python.org/3/library/functions.html#all)